scripts/init.sh

```diff
@@ -5,12 +5,10 @@ set -eux
 echo "Checkout polkadot-sdk Snowfork fork"
 pushd ..
   if [ ! -d "polkadot-sdk" ]; then
-    git clone https://github.com/Snowfork/polkadot-sdk.git
+    gitUserName=$(git config user.name)
```

Comment:
I think it's going to simpler to just pass the whole repo url, instead of re-interpreting git variables. The git user.name var doesn't necessarily map to a GitHub account.

Reply:
Also by default the paritytech polkadot-sdk repo should be cloned unless optionally overridden by a persona fork.

Reply:
Good point, changed over here: https://github.com/Snowfork/snowbridge/pull/1290/commits/13e8468e4cb51a494e54b2c4b4f55b0891f5ac26